python/tests/requirements.txt

```diff
@@ -9,4 +9,3 @@ pytest-cov==2.12.1
 pytest-benchmark==3.4.1
 pytest-flake8==1.0.7
 pytest-black==0.3.12
-gqlalchemy==1.3.2
```

Comment:
I think we need this one for e2e tests. You can rewrite `.gitmodules` to run tests from inside docker and that way we can remove this dependency similar as it is now in case of Python module unit test workflow:

```
 - name: Python modules unit tests
        if: matrix.target == 'dev'
        env:
          PYTHONPATH: "$PWD/python"
        run: |
          docker exec -i -u root ${{ env.MAGE_CONTAINER }} bash -c "cd /mage/python/ && python3 -m pytest ."
```

Reply:
but we do install it in the main requirements file, I'm not sure why it's again needed? These commands from the workflow don't install anything via pip

Reply:
I think this is resolved now